cmd/utils/flags.go

```diff
@@ -139,6 +139,10 @@ var (
 		Name:  "worker.disable",
 		Usage: "Disables worker. This flag results in block processing failure.",
 	}
+	StartBlockNumberFlag = cli.Uint64Flag{
```

Comment:
Could you add this flag to one of the flag groups?

Reply:
@aidan-kwon I will! Thanks!

Reply:
Updated!